libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp

```diff
@@ -83,7 +83,10 @@ int main(int, char**)
     test<unsigned int>();
     test_ext<unsigned int>();
     test<unsigned long>();
+    // This isn't implemented on platforms without __int128
+#ifndef _LIBCPP_HAS_NO_INT128
```

Comment:
This is wrong. The code should test for `TEST_HAS_NO_INT128`.

Reply:
> This is wrong. The code should test for `TEST_HAS_NO_INT128`.

Sure, I will do this for existing guards as well.